openassessment/assessment/test/test_staff.py

```diff
@@ -0,0 +1,348 @@
+                {"name": "yes", "points": "1", "explanation": ""},
+            ]
+        },
+        {
+            "name": "giveup",
+            "prompt": "How willing is the writer to give up the ring?",
+            "options": [
+                {
+                    "name": "unwilling",
+                    "points": "0",
+                    "explanation": "Likely to use force to keep it."
+                },
+                {
+                    "name": "reluctant",
+                    "points": "3",
+                    "explanation": "May argue, but will give it up voluntarily."
+                },
+                {
+                    "name": "eager",
+                    "points": "10",
+                    "explanation": "Happy to give it up."
+                }
+            ]
+        },
+        {
+            "name": "singing",
+            "prompt": "Did the writer break into tedious elvish lyrics?",
+            "options": [
+                {"name": "no", "points": "2", "explanation": ""},
+                {"name": "yes", "points": "0", "explanation": ""}
+            ]
+        },
+    ]
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 6 points
+ASSESSMENT_DICT = {
+    'overall_feedback': u"这是中国",
+    'criterion_feedback': {
+        "giveup": u"𝓨𝓸𝓾 𝓼𝓱𝓸𝓾𝓵𝓭𝓷'𝓽 𝓰𝓲𝓿𝓮 𝓾𝓹!"
+    },
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "no",
+        "giveup": "reluctant",
+        "singing": "no",
+    },
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 0 points
+ASSESSMENT_DICT_FAIL = {
+    'overall_feedback': u"fail",
+    'criterion_feedback': {},
+    'options_selected': {
+        "secret": "no",
+        u"ⓢⓐⓕⓔ": "no",
+        "giveup": "unwilling",
+        "singing": "yes",
+    }
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 14 points
+ASSESSMENT_DICT_PASS = {
+    'overall_feedback': u"这是中国",
+    'criterion_feedback': {},
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "yes",
+        "giveup": "eager",
+        "singing": "no",
+    }
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 14 points
+# Feedback text is one character over the limit.
+LONG_FEEDBACK_TEXT = u"是" * Assessment.MAX_FEEDBACK_SIZE + "."
+ASSESSMENT_DICT_HUGE = {
+    'overall_feedback': LONG_FEEDBACK_TEXT,
+    'criterion_feedback': {
+        "secret": LONG_FEEDBACK_TEXT,
+        u"ⓢⓐⓕⓔ": LONG_FEEDBACK_TEXT,
+        "giveup": LONG_FEEDBACK_TEXT,
+        "singing": LONG_FEEDBACK_TEXT,
+    },
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "yes",
+        "giveup": "eager",
+        "singing": "no",
+    },
+}
+
+
+
+@ddt
+class TestStaffOverwrite(CacheResetTest):
+    """
+    Tests for staff assessments made as overrides, when none is required to exist.
+    """
+
+    STEPS = ['self']
+    STEP_REQUIREMENTS = {}
+    STEP_REQUIREMENTS_WITH_STAFF = {'staff': {'required': True}}
+
+    def test_create_assessment_points(self):
```

Comment:
I'd use DDT to try out a variety of override values. In particular, it would be good to verify that staff can override to the minimum value (is that always 0?), the maximum value, and an in-between value.